shims/aggregator/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2020, NVIDIA CORPORATION.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.nvidia</groupId>
+        <artifactId>rapids-4-spark-parent</artifactId>
```

Comment:
Shouldn't the parent project be the shims project?  Applies to the other projects under the shims parent pom as well.

Reply:
I'm pretty sure that doesn't work but I'll take another look. I copied from hive shim layer, the spark shim layer doesn't actually include anything and just has submodules.  The shim/aggregator puts them all together and is the real rapids-4-spark-shims_2.12 artifact.

Reply:
I was wrong, this works.

Reply:
I did a bit of reading about it and maven terms is aggregator module vs parent.  I'm not seeing a lot of benefit one way or another here, so I went ahead and switched to the parent as I had already started